reqs/requirements-dev-snowflake.txt

```diff
@@ -1,4 +1,5 @@
 pandas<2.2.0; python_version >= "3.9"
 snowflake-connector-python>=2.5.0; python_version < "3.11"
 snowflake-connector-python>2.9.0; python_version >= "3.11" # earlier versions fail to build on 3.11
-snowflake-sqlalchemy>=1.2.3,!=1.5.2 # https://github.com/snowflakedb/snowflake-sqlalchemy/issues/485
+snowflake-sqlalchemy>=1.2.3,!=1.5.2,<1.7.0; python_version < "3.10" # https://github.com/snowflakedb/snowflake-sqlalchemy/issues/485
+snowflake-sqlalchemy>=1.2.3,!=1.5.2; python_version >= "3.10" # https://github.com/snowflakedb/snowflake-sqlalchemy/issues/485
```

Comment:
CI is currently broken due to static-analysis failing on a new version of snowflake-sqlalchemy on python 3.9